include/RAJA/policy/hip/policy.hpp

```diff
@@ -308,6 +397,7 @@ struct hip_thread_masked_loop {};
 // Operations in the included files are parametrized using the following
 // values for HIP warp size and max block size.
 //
+constexpr const RAJA::Index_type ATOMIC_DESTRUCTIVE_INTERFERENCE_SIZE = 64; // 128 on gfx90a
```

Comment:
Is this being used anywhere? Should we just use `WARP_SIZE` in its place?

Reply:
That's true its not used anywhere in this PR. I'll remove it, its a cache line size basically so its orthogonal to warp size.

Reply:
Never-mind, it is used, see https://en.cppreference.com/w/cpp/thread/hardware_destructive_interference_size.